sap-s4hana-cloud/2.6/modules/ROOT/pages/sap-s4hana-cloud-connector-reference.adoc

```diff
@@ -77,8 +77,10 @@ Basic authentication is the simplest way of enforcing access controls because it
 | Password a| String |  The account password with which to log into SAP S/4HANA. |  | x
 | Discovery Service Enabled a| Boolean |  Feature switch for automatic service discovery. | `true` |
 | Service discovery relative path a| String |  Relative path from the base URL to the endpoint used for automatic services discovery. |  `sap/opu/odata/IWFND/CATALOGSERVICE;v=0002/ServiceCollection` |s
-| Service directory a| String |  Common HTTP directory for all unlisted services. This parameter enables users to specify only service names in the unlisted services list instead of service URLs. It also enables the connector to construct the service URL using the following format: `{base URL parameter}/{this parameter}/{item from unlisted services list}` |   |
+| Scoped service discovery relative path a| String | Relative path from the base URL to the endpoint used for automatic services discovery with custom namespaces on cloud instances. |  `sap/opu/odata/IWFND/CATALOGSERVICE;v=0002/ScopedServiceCollection` | 
+| Unlisted services directory a| String |  Common HTTP directory for unlisted services and services declared in *Service Attribute* (if the service URL is not specified). This parameter enables users to specify only service names in the unlisted services list instead of service URLs or enables users to not specify a service URL in *Services Attribute*. It also enables the connector to construct the service URL using the following format: `{base URL parameter}/{this parameter}/{item from unlisted services list or service name from *Service Attribute*}` |   |
```

Comment:
parameter > field
suggestion: The "This parameter" sentence is a bit confusing. Do you need the second part of the sentence (not specify a service URL)? If they are only specifying service names, then they are not specifying a service URL.

Reply:
yes they need the second part